pxr/imaging/plugin/hdRpr/mesh.cpp

```diff
@@ -759,9 +759,9 @@ void HdRprMesh::Sync(HdSceneDelegate* sceneDelegate,
                 rprApi->SetMeshId(rprMesh, id);
             }
             for (auto& instances : m_rprMeshInstances) {
-                for (auto& rprMesh : instances) {
-                    rprApi->SetMeshId(rprMesh, id);
-                }
```

Comment:
Shouldn't we be setting object ID to the id val?

Reply:
SetMeshId incapsulates rprShapeSetObjectId method. We set prim ID as mesh object ID. And we set instanc ID as instance object ID. And RPR_AOV_MESH_ID shows base shape ID as ID of all it's instances. With RPR_AOV_OBJECT_ID all IDs are unique